setup.py

```diff
@@ -42,24 +42,11 @@
 from skbuild import setup
 from setuptools import find_packages
 
-py2 = sys.version_info[0] == 2
-py3 = sys.version_info[0] == 3
-
-py_version_old_message = "pc-ble-driver-py only supports Python version 3.6 and newer"
-requirements = []
-
-if py2:
-    print(py_version_old_message)
+if sys.version_info < (3, 6):
+    print("pc-ble-driver-py only supports Python version 3.6 and newer")
```

Comment:
This is actually wrong, it supports only to 3.9.x.
3.10.x are not supported

Reply:
@sebastianpfischer Can you elaborate a little why this is wrong? And I'ma  bit curious about what you meant by your above comment. :thinking: Is the package as a whole doesn't support 3.10.x or only this patch? :thinking:

Reply:
@codemascot, it actually got fixed in between so my previous comment  can be ignored, nevertheless, the use of “newer” for this packet is theoretically false, only up to 3.10.x is now supported : see https://github.com/NordicSemiconductor/pc-ble-driver-py/blob/master/setup.py#L119

but again, for today, technically you are right because 3.10.5 is the last python version released.